src/igvfd/metadata/constants.py

```diff
@@ -16,16 +16,30 @@
 METADATA_COLUMN_TO_FIELDS_MAPPING = OrderedDict(
     [
         ('File accession', ['files.accession']),
+        ('File id', ['files.@id']),
         ('File format', ['files.file_format']),
         ('File content', ['files.content_type']),
-        ('Accession', ['accession']),
-        ('Assay', ['assay_term.term_name']),
+        ('Fileset accession', ['accession']),  # All file_set_type accessions
+        ('Fileset Subtype', ['@type']),  # All file_set_type titles
+        ('Fileset type', ['file_set_type']),  # All assay_types file_set_type
+        ('Measurement set assay term', ['assay_term.term_name']),  # MeasurementSet assay_term
+        ('Measurement set preferred assay title', ['preferred_assay_title']),  # MeasurementSet preferred_assay_title
+        ('Analysis set assay titles', ['assay_titles']),  # AnalysisSet assay_titles
```

Comment:
You know I love comments, but I just wanted to be sure these were intentional and not accidentally left in, because they’re not on everything you added, and they closely match the human-readable titles.

Reply:
Yeah kind of it wasn't accidental as I was trying to hope in that in the future this might be of use to allow us separate out by different types. But you are right, I think the titles are now good enough. I will remove them.